document available host preflights types (#656)
replicatedhq/kurl.sh: src/markdown-pages/create-installer/host-preflights/tcp-load-balancer.md

---
path: "/docs/create-installer/host-preflights/tcp-load-balancer"
date: "2022-01-13"
weight: 36
linktitle: "TCP Load Balancer"
title: "TCP Load Balancer"
---
 

> The TCP load balancer host preflight check can be used to check and validate the connection to a specified TCP load balancer.

Reviewer: ```suggestion
The TCP load balancer host preflight check is used to check and validate the connection to a specified TCP load balancer.
```
